src/main/java/study/step5/domain/RacingGame.java

```diff
@@ -0,0 +1,25 @@
+package study.step5.domain;
+
+import java.util.Random;
+
+/*
+ * RacingGame
+ *
+ * 1.0.0
+ *
+ * 2020-11-17
+ */
+public class RacingGame {
+    private Random random = new Random();
+
+    public void start(Cars cars) {
+        for (int i = 0; i < cars.getSize(); i++) {
+            cars.getCar(i).go(randomNumber());
+        }
+    }
```

Comment:
RacingGame의 책임을 보니 게임을 진행하는 책임을 가진 것 같아요
그렇다면 RaceCount도 파라미터로 받은 다음 RacingGame이 라운드를 관리하도록 하고
Cars로 부터 car를 얻지말고 자동차들에게 움직이라고 직접 메시지를 보내면 좋을 것 같아요 :)
```suggestion
    public void start(Cars cars, int raceCount) {
        for (int i = 0; i < raceCount; i++) {
            cars.go(randomNumber());
        }
    }
```

Reply:
이해했습니다. 그런데 기회를 racingGame 쪽에서 반복하면 한게임 후, 점수view가 racingGame 안으로 들어가게 되는데요.
그럼 도메인안에서 view를 의존하게 되는거 아닌가요 :) ?

Reply:
앗 도메인으로 분리를 해주셨군요! RacingGame이 컨트롤러의 책임을 갖고 있는줄 알았습니다 ㅎㅎ;
그렇다면 제 피드백은 무시해주셔도 될 것 같아요!!

Reply:
다행이네요. 다른 피드백들은 수정하여 푸시했습니다! :)

Reply:
재요청을 주신다면 낼 점심때까지 리뷰드리겠습니다!